app/styles/_stream-widget.scss

```diff
@@ -20,3 +20,11 @@
     margin: 0;
   }
 }
+
+.stream-banner .stream-banner-recently .icon * {
+    fill: #00aeff;
+}
+
+.stream-banner .ember-basic-dropdown-content {
+    z-index: 1111;
```

Comment:
take a look at this z index manifest: https://github.com/nypublicradio/wnyc-web-client/blob/master/app/styles/_z-index.scss

would you mind replacing that raw value with a sass variable from that z index file? feel free to add a new one or re use one of the existing values.